core/sql/optimizer/NATable.cpp

```diff
@@ -5481,7 +5481,9 @@ NABoolean createNAFileSets(TrafDesc * table_desc       /*IN*/,
 
   if(postCreateNATableWarnings != preCreateNATableWarnings)
 	  tableConstructionHadWarnings_=TRUE;
-  const char *lobHdfsServer = CmpCommon::getDefaultString(LOB_HDFS_SERVER);
+  char  lobHdfsServer[256] ; // max length determined by dfs.namenode.fs-limits.max-component-length(255)
+  memset(lobHdfsServer,0,256);
+  strncpy(lobHdfsServer,CmpCommon::getDefaultString(LOB_HDFS_SERVER), strlen(CmpCommon::getDefaultString(LOB_HDFS_SERVER)));
```

Comment:
Yes the length param , I will change to 256.  That shoudl be sufficient wouldn't it. It would copy the strlen of CmpCommon::getDeafulatString(...) length upto a max of 256.  To answer Suresh,The corruption is caused not by passing in, the  lobHdfsServer.  But to be safe, I will do a strcpy into a hap allocated string within the CLI function of the passed in param. The corruption was  caused by the code a few lines below : 
ComString newSchName = "\"";
      newSchName += catalogNamePart;
      newSchName.append("\".\"");
      newSchName.append(schemaNamePart);
      newSchName += "\"";
      NABoolean lobTrace = FALSE;
The ComString's small_  data member overwrote what loHdfsServer was pointing to. So what got passed into the LOB ddl function instead of "default"  was "TRAFODION.SEABASE"! . But I can make the cli call opy over the passed in param right away.